### ISSUE TITLE
Output of `cache_check -V` does not match lvm2's _check_tool_version() parser

### ISSUE THREAD

**Issue by taoky** [jthornber/thin-provisioning-tools#294]
In [`_check_tool_version()` of lib/activate/dev_manager.c](https://github.com/lvmteam/lvm2/blob/4f9215e70f0c2c589f6d2a4ff43459901ee7d93b/lib/activate/dev_manager.c#L2422), it executes `(tool) -V` and tries to parse the version number:

```c
		if (fgets(buf, sizeof(buf) - 1, f) &&
		    (sscanf(buf, "%u.%u.%u", &v_maj, &v_min, &v_patch) == 3)) {
		// ...
		} else
			log_warn("WARNING: Cannot parse output '%s' from %s.", buf, argv[0]);
```

And this static function is used [here](https://github.com/lvmteam/lvm2/blob/4f9215e70f0c2c589f6d2a4ff43459901ee7d93b/lib/activate/dev_manager.c#L2534), when a command like `cache_check` fails.

For cache_check 0.9.x, this is correct:

```console
$ /usr/sbin/cache_check -V
0.9.0
```

But not for latest 1.0.x:

```
$ /usr/local/sbin/cache_check -V
cache_check 1.0.11
```

This could bring confusing output to users -- when their cache is corrupt they would see this instead:

```
$ sudo vgchange -ay test
  WARNING: Cannot parse output 'cache_check 1.0.11
' from /usr/local/sbin/cache_check.
  WARNING: Check is skipped, please upgrade installed version of /usr/local/sbin/cache_check!
  x logical volume(s) in volume group "test" now active
```

**Comment by mingnus** [comment]
You're right, that happens when cache_check failed. Thanks for reporting the bug. Will fix it soon.

**Comment by mingnus** [comment]
@taoky  It is fixed in v1.0.12. We can close this issue if there is no further problem. Thanks.

**Comment by taoky** [comment]
Thank you! I think that it works now.
